02.create-and-explore/05.avatars/03.create-avatars/01.avatar-standards/docs.md

```diff
@@ -41,11 +41,11 @@ High Fidelity avatars should match the following standard skeletal structure. Ea
     - LeftUpLeg
```

Comment:
Since we removed the tilde, we also need to remove the sentence higher up that says "~ indicates that a bone is optional."